x/stake/types/pool.go

```diff
@@ -103,7 +105,10 @@ func (p Pool) NextInflation(params Params) (inflation sdk.Rat) {
 	// 7% and 20%.
 
 	// (1 - bondedRatio/GoalBonded) * InflationRateChange
-	inflationRateChangePerYear := sdk.OneRat().Sub(p.BondedRatio().Quo(params.GoalBonded)).Mul(params.InflationRateChange)
+	inflationRateChangePerYear := sdk.OneRat().
+		Sub(p.BondedRatio().Round(precision).
+			Quo(params.GoalBonded)).
```

Comment:
Not super related, but we should benchmark if the order of quotient and multiplication affects performance. Such testing could be done in the rational package.